data.sql

```diff
@@ -1,7 +1,16 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-(1, 'Agumon', '2020-02-03', 0, true, '10.23' ),
-(2, 'Gabumon', '2018-11-15', 2, true, '8' ),
-(3, 'Pikachu', '2021-01-07', 1, false, '15.04' ),
-(4, 'Devimon', '2017-05-12', 5, true, '11' );
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Agumon', '2020-02-03', 0, true, '10.23' ),
+('Gabumon', '2018-11-15', 2, true, '8' ),
+('Pikachu', '2021-01-07', 1, false, '15.04' ),
+('Devimon', '2017-05-12', 5, true, '11' );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Charmander', '2020-02-08', 0, false, '11' ),
+('Plantmon', '2021-11-15', 2, true, '5.7' ),
+('Squirtle', '1993-04-02', 3, false, '12.13' ),
+('Angemon', '2005-06-12', 1, true, '45' ),
```

Comment:
- [ ] Well done on addressing most of the comments and updating your query; that's a positive step forward. However, as I mentioned in my previous review, the incorrect data was inserted into the `animals` table. Consequently, your queries are providing incorrect results, and you've included these erroneous output screenshots in the PR. I would strongly recommend that you update the inserted values, such as weights, to align with the project requirements and run your queries again. 
- [ ] If necessary, please add updated query output screenshots to the PR. This will ensure that your project meets the specified requirements accurately. Keep up the good work, and thank you for your diligence! 👍📈🐾
![image](https://github.com/anyars-encarta/vet-clinic/assets/55840999/ad0ddc4d-9568-435b-b0c0-e679d11fe34b)